src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/Version.java

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+package com.microsoft.signalr;
+
+public class Version {
```

Comment:
```suggestion
class Version {
```

Reply:
It has to be changed in the gradle task that generates this file otherwise it will just get overwritten.

Reply:
Yep, but we should change it in both.

Reply:
Yeah, changing in the other will change it here 😄

Reply:
Oh, the generated file is checked in? We probably shouldn't do that unless absolutely necessary.

Reply:
I had assumed that what was happening was the task was rewriting this file, but if it's generating it, we probably don't need to have it in Git at all